rfc/2019-12-9-pingcap-university-campus.md

```diff
@@ -0,0 +1,65 @@
+
+## Proposal
+
+The course is consists of two parts, reading materials and programming assignments. The reading materials can be collected from textbooks, slides, and notes from other related courses.
+
+### Reading Materials
+
+The list of reference for reading materials, both for database system and distributed system:
+
+- [Designing Data-Intensive Application](https://dataintensive.net/)
+- [Reading in Database system, 5th Edition](http://www.redbook.io/)
+- [Architecture of a Database system](http://db.cs.berkeley.edu/papers/fntdb07-architecture.pdf)
+- [Database System Concept](https://www.db-book.com/db6/index.html)
+- [Database System Implementation](http://infolab.stanford.edu/~ullman/dbsi.html)
+- [CMU cs15445 Database System](https://15445.courses.cs.cmu.edu/fall2019/schedule.html)
+- [CMU cs15721 Advanced Database System](https://15721.courses.cs.cmu.edu/spring2019/schedule.html)
+- [UCB cs186 course materials](https://cs186berkeley.net/)
+- [MIT 6.830 course materials](http://db.csail.mit.edu/6.830/)
+- [Stanford cs245 course materials](http://web.stanford.edu/class/cs245/)
+- [MIT 6.824 Distributed Systems](https://pdos.csail.mit.edu/6.824/)
+- [Princeton COS418 Distributed Systems](https://www.cs.princeton.edu/courses/archive/fall19/cos418/)
+- [Princeton COS316 Principle of Computer System Design](https://www.cs.princeton.edu/courses/archive/fall19/cos316/)
+- [UCB Operating Systems and System Programming](https://cs162.eecs.berkeley.edu/)
+- [Distributed System 3rd Edition](https://www.distributed-systems.net/)
+
+Many important topics have already been covered in the above. Course development participants are required to reorganize the content into slides or notes and translate them into Chinese, We hope the final content should be concrete and easy to understand, related to TiDB is preferred.
+
+### Programming Assignments
+
+For programming assignments, a distributed database system framework will be constructed. Even though there are many programming scaffolds about the database system and the distributed system, but they are not strongly connected, also C++ and Java are not easy to be mastered for young students. The new framework will be built with Golang, which is simpler and also the horsepower of the TiDB ecosystem. The content of the project will be borrowed from course materials listed below:
+
+- [CMU Bustub](https://github.com/cmu-db/bustub)
+- [UCB moocbase](https://github.com/berkeley-cs186/fa19-moocbase)
+- [MIT SimpleDB](https://github.com/MIT-DB-Class/simple-db-hw)
+- [MIT SimpleKV](https://github.com/MIT-DB-Class/simpleKV)
+
+## Rationale
+
+For TiDB Community, an educational distributed database system project in Golang and readings for novice are necessary, they are invitation cards of welcome young people to TiDB. We can open the door of the campus, and push computer system course education in universities forward.
```

Comment:
```suggestion
For TiDB Community, an educational distributed database system project in Golang for the novice is necessary. With  friendly and organized course content as invitation cards for young people to TiDB, we can open the door wider to campus, and push forward computer system course education in universities.
```

Reply:
Can you explain what is "computer system course education "

Reply:
I was thinking about "education of computer systems", I think this can be deleted, it's verbose.